### PR TITLE
feat: select custom arrow

### PR DESCRIPTION
**Pull request by atanasster**
issue: #1177 #1151

added arrow prop to `Select`

### CODE REVIEW

File: packages/components/src/Select.js [hunk 1]
```diff
@@ -34,12 +34,16 @@ export const Select = React.forwardRef((props, ref) => (
...
-    <DownArrow
-      sx={{
-        ml: -28,
-        alignSelf: 'center',
-        pointerEvents: 'none',
-      }}
-    />
+    {arrow ? (
+      arrow
+    ) : (
```
Comment (lachlanjc) [inline packages/components/src/Select.js:39]:
```suggestion
    {arrow || (
```

**Review by lachlanjc — APPROVED** [review]

File: packages/components/src/Select.js (hunk 1 above)
Comment (atanasster) [reply packages/components/src/Select.js]: @lachlanjc  thanks a lot for reviewing - I checked in changes with the simplified syntax

**Comment by hasparus** [comment]
Sweet! Could you update the changelog? (I feel we need a PR template with a todo list)

**Comment by hasparus** [comment]
What's the difference between this and #1151?

**Comment by lachlanjc** [comment]
@hasparus #1151 is the issue, this is a PR, unless I’m missing something?

**Comment by hasparus** [comment]
> @hasparus #1151 is the issue, this is a PR, unless I’m missing something?

🤦

I was so sure there's been a duplicate...

**Pull request merged**